backend/models/users.js

```diff
@@ -0,0 +1,37 @@
+const dbo = require("../db/conn");
+
+// Require hashing solution
+const bcrypt = require('bcryptjs')
+
+async function createUser(email, password) {
+    try {
+    const db = await dbo.getDb();
+    const usersCollection = db.collection('users');
+    const existingUser = await usersCollection.findOne({
+        $or: [{ email: email }],
+    });
+
+    if (existingUser) {
+        console.log('1', email)
+        console.log('Username or email already exists. Choose a different one.');
+        return;
+    }
+
+      // Hash the password
+    const hashedPassword = await bcrypt.hash(password, 12);
+
+      // Create a new user document
+    const newUser = {
+        email: email,
+        password: hashedPassword,
+    };
+
+    await usersCollection.insertOne(newUser);
+    console.log(email)
+    console.log("User account successfully created");
```

Comment:
User can either be logged in or navigated to login page, pending team's decision.

Reply:
does that mean that this would return a success status message if successful or error status if failed?

Reply:
If the user is created, the BE will continue with the post request under account.js, if the user creation has failed, it will return an error status. The final point of the user being created will be in account.js, where it will respond with a success message.